deployment/terraform/live/vars.tf

```diff
@@ -162,6 +162,11 @@ variable "refinery_google_recaptcha_secret_key" {
   default     = ""
 }
 
+variable "refinery_intro" {
+  description = "Mark-down to be displayed in the intro paragraph on the landing page."
+  default = "The refinery platform is for managing, analyzing, and visualizing data belonging to members of the [Harvard Stem Cell Institute (HSCI)](https://hsci.harvard.edu/).\s\s Users can upload various types of files, annotated with pertinent metadata, and can choose to share those files within their own lab, with collaborators. Users can also run common analyses via customizable [Galaxy](https://galaxyproject.org/) workflows and explore subsequent results with interactive visualization tools.\s\s Watch feature-specific tutorial videos below to get started. Once you have registered an account, log in, and click [My Dashboard](/dashboard) to start managing your own workspace."
```

Comment:
I think that the default here should be flip-flopped with the default being used in `config.json.erb` since cloud deployments are more often a `Stem Cell Commons` thing. Local installs would get the paragraph about: `"The refinery platform"` then.

Reply:
There is no need to provide Stem Cell Commons-specific values as defaults for Refinery. All SCC customizations are available in the `stemcellcommons` S3 bucket (`config.yaml` to be replaced by `terraform.tfvars` after v1.6.8 is released).